CMakeLists.txt

```diff
@@ -201,6 +201,7 @@ set(_acts_actsvg_version 0.4.40)
 set(_acts_autodiff_version 0.6)
 set(_acts_boost_version 1.71.0)
 set(_acts_dd4hep_version 1.21)
+set(_acts_dd4hep4ODD_version 1.27)
```

Comment:
could it make sense to set the DD4hep version to `1.28`? Version `1.27` is incompatible with python `3.12`.

Reply:
What I am doing now is directly putting the min version in the ODD (see : https://gitlab.cern.ch/acts/OpenDataDetector/-/merge_requests/92) I will then bump the ODD version in this PR and remove the extra dd4hep requirement. I think the CI images we use have at most 1.27 so I would prefer to keep the CI in line with the min version to avoid unexpected issue